generated/nifake/nifake_service.h

```diff
@@ -87,6 +87,8 @@ class NiFakeService final : public NiFake::Service {
   void Copy(const std::vector<CustomStruct>& input, google::protobuf::RepeatedPtrField<nifake_grpc::FakeCustomStruct>* output);
   std::map<std::int32_t, float> floatenum_input_map_ { {1, 3.5f},{2, 4.5f},{3, 5.5f},{4, 6.5f},{5, 7.5f}, };
   std::map<float, std::int32_t> floatenum_output_map_ { {3.5f, 1},{4.5f, 2},{5.5f, 3},{6.5f, 4},{7.5f, 5}, };
+  std::map<std::int32_t, float> nifakereal64attributevalues_input_map_ { {1, 3.5f},{2, 4.5f},{3, 5.5f},{4, 6.5f},{5, 7.5f}, };
+  std::map<float, std::int32_t> nifakereal64attributevalues_output_map_ { {3.5f, 1},{4.5f, 2},{5.5f, 3},{6.5f, 4},{7.5f, 5}, };
```

Comment:
`map` lookup on `float` is OK because we assume that they're constants in the driver API? So there's not going to be some roundoff issue?

You could also use `std::find_if` to search with a `tolerance` delegate.

Reply:
Actually, we do have few float values in DMM we didn't add because it was not supported yet.

> #define NIDMM_VAL_1_MICROAMP 0.000001
#define NIDMM_VAL_10_MICROAMP 0.00001
#define NIDMM_VAL_100_MICROAMP 0.0001
#define NIDMM_VAL_1_MILLIAMP 0.001

Should we add `tolerance` support for these roundoff issues? How would that look like? I see quite a few different ways of implementing [tolerance](https://randomascii.wordpress.com/2012/02/25/comparing-floating-point-numbers-2012-edition/).

Reply:
Just for the record, we should not be fixing the tolerance as part of this PR, which is already quite large. I do agree that with a value like `0.000001`, we will need to address it though.

Reply:
I have created [tech debt](https://dev.azure.com/ni/DevCentral/_workitems/edit/1517667/) to address float lookup scenarios. These affects only returning of enum value for floats and not returning raw value or getting enum/raw value from request objects.